R/tribble_paste.R

```diff
@@ -5,7 +5,13 @@ globalVariables(".rs.readUiPref", "datapasta") #ignore this function in R CMD ch
 #' @return The parsed table text. Useful for testing.
 #' @export
 #'
-tribble_paste <- function(){
+tribble_paste <- function(x) {
+
+  # if data is provided, try to use that to generate the tribble_paste()
+  if(!missing(x)) {
+    clipr::write_clip(x)
+  }
+
```

Comment:
Cool I had a bash earlier in the tibble_out branch. This has revealed a complication in the way I was handling NAs. For example try tribble_paste(head(airquality)) to see a bunch of NAs converted to NaNs.